.github/workflows/create-prerelease-on-tag.yml

```diff
@@ -0,0 +1,33 @@
+name: Create prerelease on tag
+on:
+  push:
+    tags:
+      - "*.*.*.*"
+  workflow_dispatch:
+jobs:
+  build-release-publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Validate manifest.json of the extension
+        uses: cardinalby/schema-validator-action@v1
+        with:
+          file: ext/manifest.json
+          schema: "https://json.schemastore.org/webextension.json"
+
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: "16.x"
+
+      - name: Lint
+        run: npm run-script build
+        shell: bash
+
+      - name: Release
+        uses: softprops/action-gh-release@v1
+        with:
+          generate_release_notes: true
+          prerelease: true
```

Comment:
Usually I like to add `draft: true` here so we can add a changelog to the release before publishing it

Reply:
Hmm:
* the changelog is going to be auto-generated thanks to `generate_release_notes` / `release.yml`
* since the `publish-firefox-development` workflow will immediately run and cause everyone with the dev version to get the update, I think it's better that it's not a draft at this point but a prerelease
* is it not possible to modify the description afterwards if there's something we don't like about the auto-generated version?

Reply:
That seems fine to me